plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-server/src/main/java/org/eclipse/che/plugin/java/plain/server/rest/ClasspathUpdaterService.java

```diff
@@ -83,6 +98,13 @@ private void updateProjectConfig(String projectWsPath)
     NewProjectConfig projectConfig =
         new NewProjectConfigImpl(
             projectWsPath, project.getName(), project.getType(), project.getSource());
-    projectManager.update(projectConfig);
+    RegisteredProject result = projectManager.update(projectConfig);
+    return CompletableFuture.completedFuture(result.getPath());
+  }
+
+  private void notifyClientOnProjectUpdate(String projectPath) {
+    List<Object> parameters = new ArrayList<>();
+    parameters.add(removePrefixUri(convertToJson(projectPath).getAsString()));
+    extensionService.executeClientCommand(CLIENT_UPDATE_ON_PROJECT_CLASSPATH_CHANGED, parameters);
```

Comment:
As for me it is better to initialize  `CLIENT_UPDATE_ON_PROJECT_CLASSPATH_CHANGED` here `che/plugins/plugin-java/che-plugin-java-ext-lang-shared/src/main/java/org/eclipse/che/ide/ext/java/shared/Constants.java`. che-ls-jdt isn't a good place because it's never used there.

Reply:
Yes, it's not used in che-ls-jdt, but still it's a good place to store the constant:
* to reserve its value for the command constant and make it visible for all the users of Commands.java
* to prevent having this command constant value from being re-defined in different plugins for different commands (which is quite possible)

IMHO